src/codewhisperer/service/inlineCompletionService.ts

```diff
@@ -109,8 +118,8 @@ export class InlineCompletionService {
                     page
                 )
                 if (RecommendationHandler.instance.checkAndResetCancellationTokens()) {
-                    // RecommendationHandler.instance.reportUserDecisionOfRecommendation(editor, -1)
-                    // vscode.commands.executeCommand('aws.codeWhisperer.refreshStatusBar')
```

Comment:
Remove these 2 lines?

Reply:
can you ellaborate? I have uncommented these two lines since there's a need to end the session(report userDecisions) whenever we get pagination cancelled.